Misc/NEWS

```diff
@@ -14,6 +14,8 @@ Core and Builtins
 
 - bpo-29546: Set the 'path' and 'name' attribute on ImportError for ``from ... import ...``.
 
+- bpo-29478: Fix passing max_line_length=None from Compat32 policy
```

Comment:
I think it would be clearer to say "If max_line_length=None is specified while using the Compat32 policy, it is no longer ignored".